app/javascript/packs/components/QuestionnaireControls.jsx

```diff
@@ -5,7 +5,7 @@
 
 import React from 'react';
 
-import { Link } from 'react-router-dom';
+import {Link} from 'react-router-dom';
```

Comment:
why are these spaces disappearing? it actually kind helps readability by having them